### DISCUSSION TITLE
What is this item?

### DISCUSSION THREAD

**Discussion by geist22** [swfans/syndwarsfx#132] [Q&A] [answered]
This is Syndicate mission 15 (Honolulu).

![15](https://github.com/swfans/swars/assets/107389189/a7fcf57f-e31f-4712-b560-c990570fcbec)

I'm not getting any credits or researchable items from it. Does picking it up even do anything?

**Comment by Moburma** [comment, marked as answer]
Not sure - briefcases don't inherently do anything unless they have the "ItemAmmo" field set, which gives you money on mission complete. This one doesn't have that set so is worth nothing, and no mission objectives tie to it, so it doesn't do anything. One thing that is interesting is that it has the State set to 6 - this generally mirrors the actual WeaponType value that ultimately decides what it is, but it doesn't here. If a few other things were set differently, 6 would mean this should actually be a nuclear grenade (which is also why it does not appear as a briefcase on the ground). So I would say either wrongly configured money briefcase (or abandoned mission objective), or maybe wrongly configured nuclear grenade.

**Reply by geist22** [reply]
I had a feeling it doesn't really do anything - if it's not money or a weapon, what else could it be?

The idea that they messed up here doesn't sound too far-fetched, as it wouldn't be the only thing in this level. Just look at the two zealots on the other side of the same building: they appear to be placed below the ground.
